### PR TITLE
Update params.md

### PR DESCRIPTION
**Pull request by ghost**
Adding vital information.

### CODE REVIEW

File: docs/csharp/language-reference/keywords/params.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ By using the `params` keyword, you can specify a [method parameter](../../../csh
  You can send a comma-separated list of arguments of the type specified in the parameter declaration or an array of arguments of the specified type. You also can send no arguments. If you send no arguments, the length of the `params` list is zero.  
   
  No additional parameters are permitted after the `params` keyword in a method declaration, and only one `params` keyword is permitted in a method declaration.  
+ 
+ The type specified in the parameter declaration must be a single dimensional array, otherwise a compiler error occurs (CS0225).
```
Comment (pkulikov) [inline docs/csharp/language-reference/keywords/params.md]: @johnnyagerard  That sentence doesn't play well with the start of the second paragraph:

>You can send a comma-separated list of arguments of the type specified in the parameter declaration or an array of arguments of the specified type.

In this topic, **type specified in the parameter declaration** refers to the type of a single parameter.

What about the following phrasing (or something around it):
>When you use the `params` keyword, you must specify a single-dimensional array of the data type in the method declaration as the example below shows.

By the way, if you want to mention the compiler error, consider adding the link to its page:
https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs0225

Using the opportunity, let me say that I like your recent contributions to the C# language reference and programming guide topics: they are good catches and definitely improve the docs.

**Comment by pkulikov** [comment]
@johnnyagerard a new version LGTM. Thanks for the update.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for adding this detail, @johnnyagerard. This small addition is a major enhancement to the params keyword. I do have a couple of suggestions for revisions.

**Comment by johnnyagerard** [comment]
@rpetrusha Code is updated as you suggested.

**Review by rpetrusha — APPROVED** [review]
Thanks for making the additional changes, @johnnyagerard. I'll merge your PR now. The changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**